src/main/java/eu/bbmri/eric/csit/service/negotiator/service/PostService.java

```diff
@@ -0,0 +1,17 @@
+package eu.bbmri.eric.csit.service.negotiator.service;
+
+import eu.bbmri.eric.csit.service.negotiator.dto.post.PostCreateDTO;
+import eu.bbmri.eric.csit.service.negotiator.dto.post.PostDTO;
+import java.util.List;
+
+public interface PostService {
```

Comment:
Please add javadoc

Reply:
Fixed in [9e79a34](https://github.com/BBMRI-ERIC/negotiator-v3/pull/50/commits/9e79a341c22c9d82b6f49666b7b974893587a33c)